dist/setup-appliance.sh

```diff
@@ -427,16 +427,34 @@ function ask {
 
 }
 ###############################################################################
+function check_service {
+    srv=$1
+    STATE=$(chkconfig $srv 2>/dev/null|awk '{print $2}')
```

Comment:
I'm wondering if we really should use `chkconfig` here. 
Nowadays all supported OS'es should have `systemctl` available.
Opinions?

Reply:
The service obssignd seems not to have been ported to systemd yet (see 2.10: https://build.opensuse.org/package/show/OBS:Server:2.10:Staging/obs-signd master: https://github.com/openSUSE/obs-sign/blob/master/dist/obs-signd.spec
), but is contained in the RECOMMENDED_SERVICES (see https://github.com/openSUSE/open-build-service/blob/master/dist/setup-appliance.sh#L447). 

After an installation of obs 2.10 SystemV services seem to be generated in systemctl
```
systemctl status obssignd
● obssignd.service - LSB: start the gpg sign daemon
   Loaded: loaded (/etc/init.d/obssignd; generated; vendor preset: disabled)
   Active: active (running) since Fri 2019-08-02 09:47:04 CEST; 2 weeks 3 days ago
     Docs: man:systemd-sysv-generator(8)
    Tasks: 3 (limit: 4915)
   CGroup: /system.slice/obssignd.service
           ├─1942 /usr/bin/perl /usr/sbin/signd -f
           └─2158 gpg-agent --homedir /srv/obs/gnupg --use-standard-socket --daemon

Aug 02 09:47:04 host systemd[1]: Starting LSB: start the gpg sign daemon...
Aug 02 09:47:04 host obssignd[1804]: Starting gpg sign daemon (signd): ..done
Aug 02 09:47:04 host systemd[1]: Started LSB: start the gpg sign daemon.
```
but I can't tell if this is always the case. It would be better to convert this service to a real systemd service first from my opinion.

Reply:
This is always the case. That said, there is a systemd service for it, it's just not installed on the openSUSE packaging (it is for the Fedora packaging).